src/frontend/index.html

```diff
@@ -54,216 +60,111 @@
-              treatise on the theory of ethics, very popular during the Renaissance. The first line
-              of Lorem Ipsum, "Lorem ipsum dolor sit amet..", comes from a line in section 1.10.32.
-              The standard chunk of Lorem Ipsum used since the 1500s is reproduced below for those
-              interested. Sections 1.10.32 and 1.10.33 from "de Finibus Bonorum et Malorum" by
-              Cicero are also reproduced in their exact original form, accompanied by English
-              versions from the 1914 translation by H. Rackham.
-            </p>
-          </section>
-        </article>
+    <div class="modal-container">
+      <div class="modal-panel">
+        <h1>Title</h1>
+        <section>
+          asdasdsa
+        </section>
       </div>
     </div>
 
-    <script>
-      // CODE TO TOGGLE THE MOBILE MENU DRAWER
-      document.querySelector('#mobile-burger').onclick = function(e) {
-        let nav = document.querySelector('nav');
-        nav.style.transform = 'translateX(-200px)';
-        nav.style.boxShadow = '0px 0px 100px var(--box-shadow-color)';
+    <!-- SCRIPTS -->
 
-        let close = document.querySelector('#mobile-burger-close');
+    <script>
+      $(() => {
+        getPostsPage(1);
+      });
+
+      function getPostsPage(page = 1) {
+        $('.content').html('loading...');
+
+        $.get(`/dummy_data/posts.json?page=${page}`)
+          .done(postsObject => {
+            const { posts } = postsObject;
+            $('.content').html('');
+            posts.forEach(post => {
+              $('.content').append(`
+                <article>
+                    <header>
+                        <h1><a title="${post.title}" href="${post.url}">${post.title}</a></h1>
+                        <div class="article-details">
+                            <div class="post-author"><a href="${post.site}">${post.author}</a></div>
+                            <div class="post-date">${post.updated}</div>
+                        </div>
+                    </header>
+        
+                    <section class="post-content">
+                        ${post.content}
+                    </section>
+                </article>
+                `);
+            });
+          })
+          .fail(error => {
+            console.error(error);
+            alert(
+              `Something went wrong.\n${error.status} ${error.statusText}\nCheck the log for more info. `
+            );
+            $('.content').html('Error loading Posts');
+          });
+      }
 
-        close.onclick = function() {
-          nav.style.transform = 'translateX(0px)';
-          nav.style.boxShadow = '0px 0px 0px var(--box-shadow-color)';
-        };
-      };
+      // Toggle the mobile menu
+      $('#mobile-burger').click(() => {
+        const nav = $('nav');
+
+        nav.css({
+          transform: 'translateX(-200px)',
+          boxShadow: '0px 0px 100px var(--box-shadow-color)',
```

Comment:
CSS in `.css` files please.  Let's use classes if possible.

Reply:
Added classes, now using jQuery .addClass and .removeClass methods to swap states.